distribution/docs/src/main/resources/_contents/_broker/route-manager-contents.adoc

```diff
@@ -0,0 +1,21 @@
+==== Route Manager
+
+===== Prerequisites for Route Manager
+
+None
+
+===== Installing and Uninstalling
+
+The Route Manager is installed as a part of the Broker App.
+
+===== Configuring
+
+Currently there are no configurations for the Route Manager.
+
+===== Using the Route Manager
+
+The Route Manager gives an administrator the ability to configure and deploy routes, queues, and topics dynamically. If a need arises for a new route, an administrator can easily develop a new route and deploy it to satisfy the requirement, rather than spending the time to develop, compile, and test new code.
+
+To deploy a new route, simply place a route `.XML` file in the `<KARAF_HOME>/routes` directory of ${branding}. To remove a route (or set of routes), delete the `.XML` file.
```

Comment:
It's `etc/routes`, right? Also I think `<${branding}_HOME>` should be used instead of `<KARAF_HOME>`.